iOS/Flows/Conversation/Conversation Header/MembersViewController.swift

```diff
@@ -43,28 +41,56 @@ class MembersViewController: DiffableCollectionViewController<MembersCollectionV
         ConversationsManager.shared.$activeConversation
             .removeDuplicates()
             .mainSink { [unowned self] conversation in
-                self.loadDataTask?.cancel()
+                self.startLoadDataTask(with: conversation)
+            }.store(in: &self.cancellables)
 
-                if let cid = conversation?.cid {
-                    self.conversationController = ConversationController.controller(cid)
-                } else {
-                    self.conversationController = nil
-                }
+        Client.shared.shouldPrintWebSocketLog = false
+        let reservationQuery = Reservation.allUnclaimedWithContactQuery()
+        let reservationSubscription = Client.shared.subscribe(reservationQuery)
+        reservationSubscription.handleEvent { [unowned self] query, event in
```

Comment:
The members vc needs to know when reservations update. If an invited user accepts their reservation then we need to remove the associated contact object and show their user object.